sapi/fpm/fpm/fpm_children.c

```diff
@@ -146,6 +146,7 @@ static struct fpm_child_s *fpm_child_find(pid_t pid) /* {{{ */
 static void fpm_child_init(struct fpm_worker_pool_s *wp) /* {{{ */
 {
 	fpm_globals.max_requests = wp->config->pm_max_requests;
+	fpm_globals.listening_socket = dup(wp->listening_socket);
```

Comment:
What's the actual point of using `dup` in this case? You could just assign it, right? It's not really an issue, just wondering if there is any point for that as I don't see any... :)

Reply:
I don't remember why, but dropping the dup() definitely doesn't work. I think the socket gets closed down the line or something.

Reply:
Ah I see - just tested and it didn't work without dup so it must be closed somewhere...